connectors/slack/template/storage.js

```diff
@@ -0,0 +1,48 @@
+const Sdk = require('@fusebit/add-on-sdk');
+
+let storage;
+module.exports.ensureStorage = (ctx) => {
+    if (!storage) {
+        storage = Sdk.getStorageClient(ctx);
+    }
+};
+
+module.exports.teamToHandler = null;
+module.exports.tokenToTeam = null;
+var mappingTimestamp;
```

Comment:
`var` -> `let`, but - is this debugging code?

Reply:
Fixed the `let`, the rest of it is used to make sure the cache is no more than 15 sec old